config/manager/manager.yaml

```diff
@@ -16,8 +16,6 @@ spec:
       labels:
         name: cluster-logging-operator
     spec:
-      securityContext:
```

Comment:
Same, I believe is still needed.  Any specific reason you see for removing?

Reply:
I had issues deploying the operator and this resolved it.  I don't see why the operator needs root access to its own file system which is what this says.  I'm open to discussion if you think otherwise.